--- xsp-1.0.6/test/Makefile.am.orig	2005-02-27 13:19:39.161389840 +0100
+++ xsp-1.0.6/test/Makefile.am	2005-02-27 13:19:55.403920600 +0100
@@ -1,13 +1,8 @@
-if NET_2_0
-SUBDIRS = 1.1 2.0
-else
 SUBDIRS = 1.1
-endif
 
 testfiles = \
 		mod-mono-server.exe.config \
 		index.aspx \
-		index2.aspx \
 		xsp.exe.config \
 		web.config \
 		global.asax \
--- xsp-1.0.6/tools/asp_state/Makefile.am.orig	2005-02-27 13:04:40.274041488 +0100
+++ xsp-1.0.6/tools/asp_state/Makefile.am	2005-02-27 13:13:32.458137208 +0100
@@ -3,13 +3,7 @@
 aspstatedir = $(libdir)/mono/1.0
 aspstate_SCRIPTS = asp-state.exe asp-state.exe.config
 
-if NET_2_0
-scripts2 = asp-state2.exe asp-state2.exe.config
-endif
-aspstate2dir = $(libdir)/mono/2.0
-aspstate2_SCRIPTS = $(scripts2)
-
-EXTRA_DIST = $(aspstate_input) asp-state.exe.config asp-state2.exe.config
+EXTRA_DIST = $(aspstate_input) asp-state.exe.config
 CLEANFILES = *.exe *.mdb
 
 aspstate_input = AssemblyInfo.cs.in \
@@ -20,6 +14,4 @@
 asp-state.exe: $(aspstate_sources)
 	$(MCS) $(MCSFLAGS) /out:$@ $^
 
-asp-state2.exe: $(aspstate_sources)
-	$(GMCS) $(MCSFLAGS) /out:$@ $^
 
--- xsp-1.0.6/tools/dbsessmgr/Makefile.am.orig	2005-02-27 13:16:33.139669440 +0100
+++ xsp-1.0.6/tools/dbsessmgr/Makefile.am	2005-02-27 13:17:10.452996952 +0100
@@ -3,13 +3,7 @@
 dbsessmgrdir = $(libdir)/mono/1.0
 dbsessmgr_SCRIPTS = dbsessmgr.exe dbsessmgr.exe.config
 
-if NET_2_0
-scripts2 = dbsessmgr2.exe dbsessmgr2.exe.config
-endif
-dbsessmgr2dir = $(libdir)/mono/2.0
-dbsessmgr2_SCRIPTS = $(scripts2)
-
-EXTRA_DIST = $(dbsessmgr_input) dbsessmgr.exe.config dbsessmgr2.exe.config
+EXTRA_DIST = $(dbsessmgr_input) dbsessmgr.exe.config
 CLEANFILES = *.exe *.mdb
 
 dbsessmgr_input = AssemblyInfo.cs.in \
@@ -20,6 +14,3 @@
 dbsessmgr.exe: $(dbsessmgr_sources)
 	$(MCS) $(MCSFLAGS) -r:System.Data.dll -out:$@ $^
 
-dbsessmgr2.exe: $(dbsessmgr_sources)
-	$(GMCS) $(MCSFLAGS) -r:System.Data.dll -out:$@ $^
-
--- xsp-1.0.6/scripts/Makefile.am.orig	2005-02-27 13:34:59.350499832 +0100
+++ xsp-1.0.6/scripts/Makefile.am	2005-02-27 13:36:08.002063208 +0100
@@ -1,32 +1,17 @@
 EXTRA_DIST = script.in
 
-bin2_scripts_real = xsp2 mod-mono-server2 asp-state2 dbsessmgr2
-if NET_2_0
-bin2_scripts = $(bin2_scripts_real)
-endif
-
 bin1_scripts = xsp mod-mono-server asp-state dbsessmgr
-bin_SCRIPTS = $(bin1_scripts) $(bin2_scripts)
+bin_SCRIPTS = $(bin1_scripts)
 
-CLEANFILES = $(bin1_scripts) $(bin2_scripts_real)
+CLEANFILES = $(bin1_scripts)
 
 plat_bindir = $(libdir)/mono/1.0
-plat_bindir2 = $(libdir)/mono/2.0
 
 REWRITE = sed \
 	-e 's,@''plat_bindir@,$(plat_bindir),g'		\
 	-e 's,@''exe_file@,$@.exe,g'
 
-REWRITE2 = sed \
-	-e 's,@''plat_bindir@,$(plat_bindir2),g'		\
-	-e 's,@''exe_file@,$@.exe,g'
-
 $(bin1_scripts): $(srcdir)/script.in Makefile.am
 	$(REWRITE) $(srcdir)/script.in > $@.tmp
 	mv $@.tmp $@
 
-if NET_2_0
-$(bin2_scripts): $(srcdir)/script.in Makefile.am
-	$(REWRITE2) $(srcdir)/script.in > $@.tmp
-	mv $@.tmp $@
-endif
